<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Receiver</class>
 <widget class="QMainWindow" name="Receiver">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Receiver{background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(255, 170, 0, 255), stop:1 rgba(255, 255, 255, 255));}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="Btn_reply">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>回复</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label3">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>收件人：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QPushButton" name="Btn_readatttach">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查看附件</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="Btn_back">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>返回</string>
        </property>
       </widget>
      </item>
      <item row="6" column="4">
       <widget class="QPushButton" name="Btn_readatttach_2">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>下载附件</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QLabel" name="label_attachname">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>1个（ImgPrecessor.png）</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label2">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>发件人：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label8">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>时  间：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="Btn_delete">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>20</height>
         </size>
        </property>
        <property name="text">
         <string>删除</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label5">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>附  件：</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="7" column="0" colspan="6">
       <widget class="QTextBrowser" name="textBrowser">
        <property name="styleSheet">
         <string notr="true">border-image: url(:/background/bg_receivertoum.png);

</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="5">
       <widget class="QLabel" name="label_reciever">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>xiaoyue_xie&lt;xiaoyue_xie@126.com&gt;</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="5">
       <widget class="QLabel" name="label_time">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>2019年07月01日 18:03 </string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="5">
       <widget class="QLabel" name="label_sender">
        <property name="font">
         <font>
          <family>新宋体</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>xiexy&lt;xiexy@nwafu.edu.cn&gt;</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="6">
       <widget class="QLabel" name="label_Title">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>ImgProcrssor</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>875</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
